src/backend/joanie/core/api/client/__init__.py

```diff
@@ -541,6 +542,50 @@ def withdraw(self, request, pk=None):  # pylint: disable=no-self-use, invalid-na
 
         return Response(status=HTTPStatus.NO_CONTENT)
 
+    @action(detail=True, methods=["POST"])
+    def pay_failed_installment_payment(self, request, pk=None):  # pylint: disable=no-self-use, invalid-name, unused-argument
```

Comment:
```suggestion
    def pay_failed_installment_payment(self, request, pk=None):  # pylint: disable=invalid-name, unused-argument
```
you are using `self`. Why the name is invalid ?

Reply:
Alright, we can take it away. But it seems like a bunch of other endpoints needs some cleaning one day.

Reply:
New version and pylint is okay with that : 
```suggestion
    def submit_installment_payment(self, request, pk=None):  # pylint: disable=unused-argument
```